webapp/src/main/webapp/js/individual/individualQtipBubble.js

```diff
@@ -6,48 +6,44 @@ $(document).ready(function(){
     
     $.extend(this, i18nStrings);
 
+    $('head').append('<style id="researchAreaCSS">.qtip { font-size: 14px; max-width: none !important; } .researchAreaTip { background: url(' + imagesPath + '/individual/researchAreaBubble.png) no-repeat; } </style>');
+    padding: '0em',
```

Comment:
`padding: '0em',` in particular? I don't believe that's a valid standalone javascript statement and it's not contained inside of any qTip or function declaration. It looks like it either needs to be put in the qtip styling for `researchAreaCSS` or moved down to the styling for the callback provided to `$('#researchAreaIcon').each`.